software-development-observability-terraform/sumologic_collection.tf

```diff
@@ -0,0 +1,270 @@
+# This script creates Sumo Logic Collector and Sources.
```

Comment:
May want to break up into a separate doc for setting up content

Reply:
renamed